components/chainhook-event-observer/src/chainhooks/types.rs

```diff
@@ -24,6 +25,30 @@ impl HookFormation {
         }
     }
 
+    pub fn serialized_stacks_predicates(
```

Comment:
Should it be `.serialize_` instead of `.serialized_`?

Reply:
Ended up with `get_serialized_stacks_predicates`